index.d.ts

```diff
@@ -222,9 +12,9 @@ declare namespace ansiStyles {
 	}
 
 	interface ColorBase {
-		readonly ansi: ColorConvert;
-		readonly ansi256: ColorConvert;
-		readonly ansi16m: ColorConvert;
+		ansi256(ansi256: number): string;
+
+		ansi16m(rgb: readonly [number, number, number]): string;
```

Comment:
Why a tuple and not just multiple arguments? If the argument is because the color conversion returns that, the user could just use spreading.